### DISCUSSION TITLE
Grafana-Agent Service Logs Location

### DISCUSSION THREAD

**Discussion by v15294** [grafana/grafana#64777] [Observability]
Hello, I had a question regarding accessing Grafana-Agent logs on my centos7 machine that I’m testing. My grafana-agent version 0.30.02. My agent is installed as a service, so I can check systemctl status grafana-agent and check that it’s working and view metrics on a Grafana dashboard. I’d like to see how I can access the logs of the service itself. From looking around extensively, I’ve seen some answers like /var/log/grafana/grafana.log or it’s only sending it to stdout. I also checked /var/log/messages but not seeing any information there for my agent also. My grafana-agent configuration itself has log level as debug and can see all the information when doing journalctl. I’m just trying to redirect that output of the agent service to a file (var/log/grafana.log). I’m trying to view is the logs of the agent itself in a file rather than journalctl. Please let me know if I could provide any more information, haven’t been able to find much in the documentation. Appreciate any help.

**Comment by v15294** [comment]
Resolved

**Comment by rthomasson** [comment]
Please take me off this distribution list.

From: v15294 ***@***.***>
Reply-To: grafana/grafana ***@***.***>
Date: Wednesday, March 15, 2023 at 6:25 PM
To: grafana/grafana ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [grafana/grafana] Grafana-Agent Service Logs Location (Discussion #64777)


Resolved

—
Reply to this email directly, view it on GitHub<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fgrafana%2Fgrafana%2Fdiscussions%2F64777%23discussioncomment-5327103&data=05%7C01%7Crthomasson%40cassidy.com%7C584ae3d32de841d84c1a08db25a43e8f%7C54247946c4cc4f10a9449656acacb39b%7C0%7C0%7C638145159583945481%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=dsM6q9ap7uxq6JbHCiNLCoqx38H6tT3y%2F92A%2Fm%2BQ2Y8%3D&reserved=0>, or unsubscribe<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAAJA6TNN2KXMMLKMM4PIKFLW4I3S3ANCNFSM6AAAAAAV3BCE4Q&data=05%7C01%7Crthomasson%40cassidy.com%7C584ae3d32de841d84c1a08db25a43e8f%7C54247946c4cc4f10a9449656acacb39b%7C0%7C0%7C638145159583945481%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=79tyWMOpvHMMmg%2BjwlC%2BdENeC011ifldrd%2Fo66KTTPI%3D&reserved=0>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

CAUTION: This email originated from outside the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.

**Comment by rthomasson** [comment]
Randy, can you help me get off this list?  I am not sure why I am getting these emails, but I am getting hundreds of them everyday.   I don’t have even have a github account.  Thanks. Russ

From: Randy Thomasson ***@***.***>
Reply-To: grafana/grafana ***@***.***>
Date: Wednesday, March 15, 2023 at 6:41 PM
To: grafana/grafana ***@***.***>
Cc: Randy Thomasson ***@***.***>, Your activity ***@***.***>
Subject: Re: [grafana/grafana] Grafana-Agent Service Logs Location (Discussion #64777)

Please take me off this distribution list.

From: v15294 ***@***.***>
Reply-To: grafana/grafana ***@***.***>
Date: Wednesday, March 15, 2023 at 6:25 PM
To: grafana/grafana ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [grafana/grafana] Grafana-Agent Service Logs Location (Discussion #64777)


Resolved

—
Reply to this email directly, view it on GitHub<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fgrafana%2Fgrafana%2Fdiscussions%2F64777%23discussioncomment-5327103&data=05%7C01%7Crthomasson%40cassidy.com%7C584ae3d32de841d84c1a08db25a43e8f%7C54247946c4cc4f10a9449656acacb39b%7C0%7C0%7C638145159583945481%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=dsM6q9ap7uxq6JbHCiNLCoqx38H6tT3y%2F92A%2Fm%2BQ2Y8%3D&reserved=0&<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fatpscan.global.hornetsecurity.com%2Findex.php%3Fatp_str%3DZYNC5MY4aFjKQ0CIwTnUX3QUsqCQ-AOxqBJMaBXOOEfJplbder8SLO-ZXjr4w_XJEk9u99AuQ22x6kX5bKpdKizCS9quy8zw-wEwevwCTkC0xz_HR7zXRFGP7qAT7U9RlVXBO_5CGXQHG60BdXjM_OTmWlZY8j8OvV8-c9OblTo81vGuIy32tJiqlBpaZu-aOg6jNeDdh9dtK5U2c-4eKdGp1_tNbpBSjE43dEaa5xp4KLzlknOV_hc4zZkuH71eITvIJqJ6WRKVpk1ibBs6uXpCP9ElQLVLZKUrHO3AJpDlpW17bXZPb9muTTyeFRpBy6YdHnDNXOOwYcKLKeCnEYJ4SDDSQo-9AS2T9X5XO0iRL_OEzkVwXdFy9JPahPv2rSpZrMeoSIl1uDvIP6ECTHVArhebywMycclypvoTqYW6OnI6Tikmpv7zGQ8ijB7G-cDVx5_tP28FpNxSehyxdomgx8Jlrp3MNYlYessDKgeg7sssFC3hUhA9kVppCOp8ZifOn98BWnP3L__ajyJQI33fDEcTRQDfS7mYaghbwKzKxAAVJG1hxp1o8EPyOOmmVOJRANE5iZcXFR7IaxgqiOI2xteBftUibY0la1fX0jDwdbsvjoNyJRHrAB8U9EMp9661XrqdCTToemS6Fp2Q1v8XTG_GY4Q6eo78xBPU7c32UVrkfY4JBUorJ8gLEFEc8rimrAphzLjMk4f0BLCWtJ3JZ5cQUZpCucjFvwi407cHR0MqXrn4Mzcs9ZUtrYsr37d_yXC5fklFRdO3eZnQH5Fc6uYlIh_tPybBAx30OIyjHRZNrsuBH0EsSdrRd4YZBdvuFyef32QbKdHcTdcSC9Zc_fd-3aHYgnFVEB0ea9BeDpdM18g78jYDWMZvdiL3AZRv5xN4iW_-gNquQ6lzs3bApKMYdICfOQmk3TcpPLGCygHV1gdOh1oNVOFj6toFUzIZtbbXQB5XbVgjOjojZRI6uXlNDJbc_EkZIzo6I6oOD5Ad9f6gzq6RKGL2_h4&data=05%7C01%7Crthomasson%40cassidy.com%7Cb870a141e6a24b1b6a5808db25a661e2%7C54247946c4cc4f10a9449656acacb39b%7C0%7C0%7C638145168765318546%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=i3MMeh0iExWks6pTqdjP6qPgZAeGiQyXjS%2Bk3aG7iEM%3D&reserved=0>gt;, or unsubscribe<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAAJA6TNN2KXMMLKMM4PIKFLW4I3S3ANCNFSM6AAAAAAV3BCE4Q&data=05%7C01%7Crthomasson%40cassidy.com%7C584ae3d32de841d84c1a08db25a43e8f%7C54247946c4cc4f10a9449656acacb39b%7C0%7C0%7C638145159583945481%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=79tyWMOpvHMMmg%2BjwlC%2BdENeC011ifldrd%2Fo66KTTPI%3D&reserved=0&<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fatpscan.global.hornetsecurity.com%2Findex.php%3Fatp_str%3DfXK2zVylh4ij6LxcL7F8hkOTGa6A229gAVRqHD9tErouDtf8DbDHvtQ_j1Z6xu1d1iOCUOobjzxyId1_J43oPrnFccR6MWO_RpsqgMm7DTFm4ZAaHfsCM33rXHbNwOaV35jusS5KWCa5N9nyS8B3TARAyTrb9mYX4EylTElNSoqLGIiuKJSdMMiDgdPHOKBl2ujUv5IJmC80ptIysOvmR-gKf0Kr2kUiq15qULXiqoKG-RcuhUfrKxpmSAlNsZ4F4hbV-xPn1pgvnBVH0BvoT2mc9EMjrUEyWFkoAdjHFUXQ7zECTLDKgTLQ5qZVn3tEKno3qiBe6FOqY0jbtVnubp61npCCQa79JrOTmM4qTRZur4eTBCwxCg1jjA5xjmQ8dX00ECmJtDGUfRYPScFMwpabeS8IBlwsILmLh1KIm3xqrhwOQbpNGslZxdWWS-1ndm3MHnOmPuC6jd81Pqaot2M7WtTUtBWte4kQmuqNhn8OnG1lYgOHYFxp00T1DOfpcAsnLcEQSyQ-FxAVYb_4a_U_xgOF2lImFK711ZPTkkcJA7_JTWC8T3hVM3j57iyCt1v-L87XBeiIJWyMLpWGGBBLLONLyapNa_mY3R7HvPaZg2ZKAq-I1g--N7lfmBQnKyRy2oBBIlPKmr-1mvdTGdSAQpB85qdFlUY6KKw_K5ECXiudfFOXsNOqwUlnBz0D-1_JgZmjm9i6DW2NprRJfzdD8Du9WnlYgajsLSzOgViGroVmw13oBh8kS6Wy9nCUA26Ddf-Hbstj-BtGKs_vsjujZemjswbi7Lwk1eh4oR40BnO5sMIQ9QuQhEUWdrR_k0iQUdPxYDHt_fj0-n_7_ogaAgbCYZckantlvFXiu81fMBkkC-KEgT-1Ze_dTNomFRLsw9YkTx3UGuIh1GePbUC9pK8bBXWHr11Nix8t_3GFnnSVbsulf6IFilLpClQn_Sk3RHO-vm6YQ3HxNRebZ8dl0pobmR2AztXdgPEjOjojGQ8JFyk5U1NDWaKqIzo6IzvlIOrgdQ_KQDDBlsL-TqI&data=05%7C01%7Crthomasson%40cassidy.com%7Cb870a141e6a24b1b6a5808db25a661e2%7C54247946c4cc4f10a9449656acacb39b%7C0%7C0%7C638145168765318546%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=LZTS%2FcZxt5tamBIy1K%2Fx2i1z%2F%2BwaN6IsvwB76D19oI8%3D&reserved=0>gt;.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

CAUTION: This email originated from outside the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.



—
Reply to this email directly, view it on GitHub<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fgrafana%2Fgrafana%2Fdiscussions%2F64777%23discussioncomment-5327333&data=05%7C01%7Crthomasson%40cassidy.com%7Cb870a141e6a24b1b6a5808db25a661e2%7C54247946c4cc4f10a9449656acacb39b%7C0%7C0%7C638145168765318546%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=pBMYmRTL6sIMOG0qRob%2F4C8fHcgT%2FDHA0p1Sjr1lL3M%3D&reserved=0>, or unsubscribe<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAAJA6TLSRE6FYEMWLOTMPL3W4JAPLANCNFSM6AAAAAAV3BCE4Q&data=05%7C01%7Crthomasson%40cassidy.com%7Cb870a141e6a24b1b6a5808db25a661e2%7C54247946c4cc4f10a9449656acacb39b%7C0%7C0%7C638145168765318546%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=VYNClzhaOBIH1qakBlVqVwszyDR36PJ7kKloP55AFrc%3D&reserved=0>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

CAUTION: This email originated from outside the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.

**Comment by gelicia** [comment]
This was resolved, so I will close the discussion.
